src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -379,6 +423,102 @@ public static string GetModifiedJsonString(string[] reps, string enumString)
 ";
         }
 
+        /// <summary>
+        /// Config containing environment variables.
+        /// </summary>
+        private static string _configWithEnvVar = @"
+          {
+            ""$schema"": ""/dab-schema.json"",
+            ""data-source"": {
+              ""database-type"": ""cosmosdb_nosql"",
+              ""options"": {
+                ""database"": ""@env('DATABASE_NAME')"",
+                ""schema"": ""@env('GRAPHQL_SCHEMA_PATH')""
```

Comment:
because you have a new DataSource converter, you'll need tests to validate behavior of:
- `options.database` and `options.schema` are **null** because this has caused regressions in the past.
- `options` is null or empty.